json_ai_files/save_load_details.json

```diff
@@ -401,6 +401,24 @@
                 "nem_ap_val": "1",
                 "special_movement": "none"
             }
+        },
+        "3": {
+            "True": {

```

Comment:
This file being updated without also including the updated saves results in the charges in this file not being useful to anyone other than the person who updated this file. Please include the relevant saves

Reply:
I'll add those directly.

Reply:
Should be updated now.